backend/src/api/bitcoin/esplora-api.interface.ts

```diff
@@ -6,6 +6,7 @@ export namespace IEsploraApi {
     size: number;
     weight: number;
     fee: number;
+    sigops?: number;
```

Comment:
Why is this optional? Once merged in Electrs, it will always return the sigops.

Reply:
It's needed for compatibility with other backends, since Core, romanz/electrs, older esplora versions etc still won't supply sigops.

Ideally we'd just fill in the missing value using the local calculation, but we aren't guaranteed to have prevouts available everywhere the interface is instantiated.

Reply:
Oh yeah, heh.